ads/google/a4a/utils.js

```diff
@@ -914,3 +918,17 @@ export function getAmpRuntimeTypeParameter(win) {
   const art = getBinaryTypeNumericalCode(getBinaryType(win));
   return isCdnProxy(win) && art != '0' ? art : null;
 }
+
+/**
+ * Returns boolean whether the page is canonical AMP, based on the url
+ * of the page. If we are testing locally, always returns that we are
+ * not on canonical.
+ * @param {!Window} win
+ * @return {boolean}
+ */
+export function isCanonical(win) {
```

Comment:
Replace with isCdnProxy (not of it) created in #15421

Reply:
done